src/main/plugin/iso19139.ca.HNAP/layout/layout.xsl

```diff
@@ -225,13 +231,19 @@
           <xsl:for-each select="$metadataOtherLanguages/lang">
             <xsl:variable name="code" select="@code"/>
             <xsl:variable name="currentLanguageId" select="@id"/>
+            <xsl:variable name="currentLanguageId_none_ISO">
```

Comment:
Odd to use none_ISO for a variable name.  Why not rename the other variable to ISO?

Reply:
I have rename the variable to currentLanguageId_iso639_2t to indicate it is iso639-2t format.